src/main/resources/application.yml

```diff
@@ -17,6 +17,52 @@ spring:
     group:
       test:
         - "stdout"
+      dev:
+        - "stdout"
+
+  security:
+    oauth2:
+      resourceserver:
+        jwt:
+          jwk-set-uri: ${api.base.url.oauth}/.well-known/jwks.json
+
+      client:
```

Comment:
not sure we need this part, would we ever be making API calls?

Reply:
good point, will remove